### DISCUSSION TITLE
custom elevation model + tiled texture

### DISCUSSION THREAD

**Discussion by radoslavirha** [visgl/deck.gl#5665] [Q&A]
Hi, I'm trying to create TerrainLayer with custom elevation model (not tiled) and texture (openstreet map, in future orthophoto),
but {x} {y} {z} from texture url is not parsed. Is it possible what I want to achieve? Thanks
![image](https://user-images.githubusercontent.com/23722745/114705224-cb001e00-9d27-11eb-9250-3a492f83ec66.png)

**Comment by kylebarron** [comment]
The terrain layer has either tiled or non-tiled modes. Your clearest options to me are either: tile your custom elevation model and use the `TerrainLayer`'s tiled mode, or create a single texture to overlay and use its non-tiled mode. I.e. join together OSM tiles to match the size of your elevation model.
